sources/web/datalab/static/util.js

```diff
@@ -1,7 +1,14 @@
 define(() => {
+
+  window.datalab = window.datalab || {};
+  window.datalab.debug = false;
```

Comment:
You type `window.datalab.debug = true` in your browser console. It's not ideal, but a good fix until we figure out a build flag or something more proper.